src/App.tsx

```diff
@@ -1,26 +1,296 @@
+              ref={newTodoInputRef}
+              data-cy="NewTodoField"
+              type="text"
+              className="todoapp__new-todo"
+              placeholder="What needs to be done?"
+              value={newTodoTitle}
+              onChange={e => setNewTodoTitle(e.target.value)}
+              disabled={isInputDisabled}
+            />
+          </form>
+        </header>
+
+        <section className="todoapp__main" data-cy="TodoList">
+          {filteredTodos.map(todo => (
+            <div
+              data-cy="Todo"
+              className={`todo ${todo.completed ? 'completed' : ''}`}
+              key={todo.id}
+            >
+              <label htmlFor="title" className="todo__status-label">
+                {' '}
+                <input
+                  id="title"
+                  data-cy="TodoStatus"
+                  type="checkbox"
+                  className="todo__status"
+                  checked={todo.completed}
+                />
+              </label>
+
+              <span data-cy="TodoTitle" className="todo__title">
+                {todo.title}
+              </span>
+
+              <button
+                type="button"
+                className="todo__remove"
+                data-cy="TodoDelete"
+                onClick={() => deleteTodo(todo.id)}
+              >
+                ×
+              </button>
+
+              <div
+                data-cy="TodoLoader"
+                key={todo.id}
+                className={`modal overlay ${loadingTodoId === todo.id ? 'is-active' : ''}`}
+              >
+                <div className="modal-background has-background-white-ter" />
+                <div className="loader" />
+              </div>
+            </div>
+          ))}
+        </section>
+
+        {todos.length > 0 && (
+          <footer className="todoapp__footer" data-cy="Footer">
+            <span className="todo-count" data-cy="TodosCounter">
+              {counter} items left
+            </span>
+
+            <nav className="filter" data-cy="Filter">
+              <a
+                href="#/"
+                className={`filter__link ${statusFilter === 'All' ? 'selected' : ''}`}
+                data-cy="FilterLinkAll"
+                onClick={() => setStatusFilter('All')}
+              >
+                All
+              </a>
+
+              <a
+                href="#/active"
+                className={`filter__link ${statusFilter === 'Active' ? 'selected' : ''}`}
+                data-cy="FilterLinkActive"
+                onClick={() => setStatusFilter('Active')}
+              >
+                Active
+              </a>
+
+              <a
+                href="#/completed"
+                className={`filter__link ${statusFilter === 'Completed' ? 'selected' : ''}`}
+                data-cy="FilterLinkCompleted"
+                onClick={() => setStatusFilter('Completed')}
+              >
+                Completed
+              </a>
```

Comment:
Use `Object.value(your created enum)` and render these options with `map()` method